methods/boin/boinsim.R

```diff
@@ -0,0 +1,144 @@
+# required packages
+
+library(magrittr)
+library(escalation)
+library(ggplot2)
+
+# lists
+
+n_sims <- list()
+
+model <- list()
+sims <- list()
+
+selection <- list()
+selection_df <- list()
+
+treated <- list()
+treatedpct <- list()
+treatedpct_df <- list()
+
+dist_accuracy <- list()
+mean_accuracy <- list()
+
+dist_overdose <- list()
+mean_overdose <- list()
+
+dist_length <- list()
+mean_length <- list()
+
+# variables needed (UI)
+
+n_doses <- 5
+ttl <- 1/3
+max_n <- 24
+start_dose <- 1
+current_seed <- 12345
+n_sims$boin <- 100
+true_dlt_ss <- c(0.05, 0.15, 1/3, 0.5, 0.8)
+cohort_size <- 3
+stop_n_mtd <- 15
+
+## NOT YET COLLECTED IN Q_DATABASE
+p_tox <- 0.6
+p_saf <- 0.1
+use_stopping_rule <- TRUE
+n_cohorts <- 10
```

Comment:
This all should already be in the current q_database on main right?

Reply:
Yes this is all implemented in the method_q_database. I haven't generated any new dummy data since including BOIN so just listed them in the script temporarily.